From e4d254a33f6c222afba8c6e05d733f1f7856bb93 Mon Sep 17 00:00:00 2001
From: Allen Xu <b45815@freescale.com>
Date: Sat, 22 Nov 2014 13:15:19 -0600
Subject: [PATCH 0726/5242] MLK-9909 ARM: dts: add imx6sx sdb revA QSPI legacy
 support

commit  3f2e8b963956804b3c467a050f015769a3b2a8ee from
https://source.codeaurora.org/external/imx/linux-imx.git

add sdb revA board QSPI legacy support for Spansion QSPI chip.

Signed-off-by: Allen Xu <b45815@freescale.com>
(cherry picked and merge from commit 3fd143ae78007e58d1dc45bc829b462a12b93cd2)
(cherry picked from commit 8317ae5998a20b3303a2a9209c29ab5674e09dc3)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb.dts |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dts b/arch/arm/boot/dts/imx6sx-sdb.dts
index 5d7a32f..fa2cbdc 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb.dts
@@ -34,6 +34,7 @@
 	pinctrl-0 = <&pinctrl_qspi2>;
 	status = "okay";
 
+#ifndef SPANSIONFLASH
 	flash0: n25q256a@0 {
 		#address-cells = <1>;
 		#size-cells = <1>;
@@ -49,6 +50,7 @@
 		spi-max-frequency = <29000000>;
 		reg = <1>;
 	};
+#endif
 };
 
 &reg_arm {
-- 
1.7.9.5

